src/skin.cpp

```diff
@@ -446,6 +446,7 @@ void SkinInfillAreaComputation::generateInfill(SliceLayerPart& part, const Polyg
     else
     {
         part.infill_area = final_infill;
+        part.infill_area = final_infill;
```

Comment:
Looks like a mistake; why is this here twice?

Reply:
Yeah. fixed.